NuCivic/dkan
# More informative error message on duplicate path alias CIVIC-4385

Issue: CIVIC-4385

## Description

If you create a node with titles that result in the exact same path alias, en error is returned: "The alias is already in use." This may be more of an Enhancement Request, but it isn't clear if this was intended. Currently the error is not understood by non-technical users and appears broken to them.

## How to reproduce

- Create a dataset
- Create another dataset with the same title
- You will see the error

## Acceptance Criteria
Make the error much easier to understand, with descriptive information for non-technical users to fix on their own

## Reminders
